lib/web_console/request.rb

```diff
@@ -10,6 +10,9 @@ class Request < ActionDispatch::Request
     cattr_accessor :whitelisted_ips
     @@whitelisted_ips = Whitelist.new
 
+    # Define a vendor MIME type. We can call it using Mime::WEB_CONSOLE constant.
+    Mime::Type.register 'application/vnd.web-console.v2', :web_console
```

Comment:
We can actually name this `web_console_v2`, so we can respond to old version and act upon them. Say, we currently accept `application/vnd.web-console.v3`, but someone sends `application/vnd.web-console.v2`. We can explicitly tell the users to upgrade their browser extensions.